Dockerfile

```diff
@@ -1,18 +1,19 @@
 # A minimal Nginx container including ContainerPilot
-FROM nginx:1.11
+FROM nginx:1.13
 
 # Add some stuff via apt-get
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         bc \
+        ca-certificates \
         curl \
         unzip \
     && rm -rf /var/lib/apt/lists/*
 
 # Install Consul
 # Releases at https://releases.hashicorp.com/consul
-RUN export CONSUL_VERSION=0.7.3 \
-    && export CONSUL_CHECKSUM=901a3796b645c3ce3853d5160080217a10ad8d9bd8356d0b73fcd6bc078b7f82 \
+RUN export CONSUL_VERSION=0.7.5 \
```

Comment:
The next version should probably jump to Consul 0.8.x, but I'm stay on the 0.7 branch because of some bugs in the first releases of 0.8.